TwitterClone/TwitterClone/settings.py

```diff
@@ -131,14 +130,11 @@
 # https://docs.djangoproject.com/en/3.1/howto/static-files/
 
 STATIC_URL = '/static/'
-STATIC_ROOT = "sub_twitter/static"
-STATICFILES_DIRS = [
-    os.path.join(BASE_DIR,'static')
-]
+STATIC_ROOT = "static"
```

Comment:
Just a simple `STATIC_ROOT` should be enough, only when your project grows bigger you will need to use finer settings like `STATICFILES_DIRS` etc